index.html

```diff
@@ -150,11 +155,124 @@ <h1 class="greet">
             </ul>
           </div>
           <div class="see-project">
-            <button class="button">See Project</button>
+            <button type="button" class="button">See Project</button>
           </div>
       </div>
       </section>
-      <div class="aboutme_container">
+      <!----------------------------Desktop--------------------------------->
+      <div class="desktop-work">
+        <div class="desktop_projects" id="desktop_1">
+          <div class="work-img">
+            <img src="assets/Snapshoot Portfoliodesktop.png">
+          </div>
+          <div class="desktop_info">
+            <div class="project-title">Tonic</div>
+            <div class="some-list">
+              <img src="assets/Frame 2.png">
+            </div>
+            <div class="work-info">
+              A daily selection of private and personalised<br>
+              reads ; no accounts<br>
+              or sign-ups required.
+            </div>
+            <div class="langs">
+              <ul class="lang-list">
+                <li class="Html">html</li>
+                <li class="ruby">Ruby on rails</li>
+                <li class="Css">css</li>
+                <li class="js">javascript</li>
+              </ul>
+            </div>
+            <div class="see-project">
+              <button type="button" class="button">See Project</button>
+            </div>
+          </div>
+        </div>
```

Comment:
- [ ] You should stick to the design as much as possible. Please adjust the spacing from left and right, also if possible please fix the text as it covers the whole width as shown in the image below, please fix the buttons to match the Figma design.
- [ ] Please make sure that the page is divided into half of it containing the project's picture and the other half containing the project's description.
- [ ] Every cards have the same issue please fix the same issue to all the cards.
### Your design
![his](https://user-images.githubusercontent.com/81727988/213866517-af61ae9c-1305-43ca-a0be-4716d116e2bc.jpg)

### Figma design
![figma](https://user-images.githubusercontent.com/81727988/213866524-a1056e6b-eb10-4544-9b0a-9d93a3b3cc82.jpg)